src/library/Box/Tools.php

```diff
@@ -406,8 +406,11 @@ public function updateConfig()
         $newConfig['maintenance_mode']['allowed_ips'] ??= [];
 
         $newConfig['disable_auto_cron'] ??= false;
-        $newConfig['locale_date_format'] = ($currentConfig['locale_date_format'] === '%A, %d %B %G') ? 'l, d F o' : $currentConfig['locale_date_format'];
-        $newConfig['locale_time_format'] = ($currentConfig['locale_time_format'] === ' %T') ? ' G:i:s' : $currentConfig['locale_time_format'];
+
+        $newConfig['i18n']['locale'] ??= 'en_US';
```

Comment:
We should be bringing over the old values for these when they exist

Reply:
Thanks for pointing this out - updated in latest commit. 🙂